config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root to: "users#index"
+  get '/users/:id' , to: 'users#show'
+  get '/users/:id/posts' , to: 'posts#index'
+  get '/users/:id/posts/:post_id' , to: 'posts#show'
```

Comment:
- [ ] [optional] what you did it'll generate the right routes but it's a little bit odd cause imagine you have a big app with 50 routes so please it would be better to use `resources` in the order you can achieve what you did whit only two lines of code like this 
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```